www/data/showcase.json

```diff
@@ -229,6 +229,12 @@
     "github": "xijaja/fresh-notion-blog",
     "image": "fresh-notion-blog"
   },
+  {
+    "title": "Deno Paste",
+    "link": "https://deno-paste.deno.dev/",
+    "github": "stephenmelnicki/fresh",
```

Comment:
Is `stephenmelnicki/deno_paste` the right repository?

Reply:
Oh shoot! I must've missed this. I'll get this updated in just a moment.

Good catch! Thank you!

Reply:
And [that](https://github.com/denoland/fresh/pull/987/commits/b10fd25a539ec8131a45465a0c88970624bd5dc7) should do the trick.

Thanks again!